code/modules/mob/living/carbon/human/species/outsider/revenant.dm

```diff
@@ -1,3 +1,6 @@
+/mob/living/carbon/human/revenant
+	universal_understand = TRUE
```

Comment:
This isn't the way to do this. This won't work if you spawn a human and then change the species.